sync.py

```diff
@@ -143,9 +143,14 @@
 
 async def main(args):
     semaphore = asyncio.BoundedSemaphore(args.limit)
-    async with aiohttp.ClientSession() as session:
-        await upload_scripts(session, args.url, args.username, args.password, semaphore)
-        await upload_extension_attributes(session, args.url, args.username, args.password, semaphore)
+    if args.allowinvalidssl:
+        async with aiohttp.ClientSession(connector=aiohttp.TCPConnector(verify_ssl=False)) as session:
```

Comment:
We can remove the if statement and just run these lines right here.

Reply:
I think i'm more confused. Sorry
`async def main(args):
    semaphore = asyncio.BoundedSemaphore(args.limit)
    async with aiohttp.ClientSession() as session:
        await upload_scripts(session, args.url, args.username, args.password, semaphore)
        await upload_extension_attributes(session, args.url, args.username, args.password, semaphore)
    if args.allowinvalidssl:
        async with aiohttp.ClientSession(connector=aiohttp.TCPConnector(verify_ssl=False)) as session:
            await upload_scripts(session, args.url, args.username, args.password, semaphore)
            await upload_extension_attributes(session, args.url, args.username, args.password, semaphore)`
?

Reply:
We should be able to:

```
async with aiohttp.ClientSession(connector=aiohttp.TCPConnector(verify_ssl=False)) as session:
    await upload_scripts(session, args.url, args.username, args.password, semaphore)
    await upload_extension_attributes(session, args.url, args.username, args.password, semaphore)

Reply:
Ahh... Okay, so I have a reason for preferring not a hard switch in the code like that.

In my environment, our DEV environment through a ci runner can have invalid ssl. Its a test environment torn away, and may have a self signed cert.

Production on the other hand, I want to verify that certificate. I want to only post to a server with a valid SSL config. Ergo, running the command as part of my runner, i can flag it off for dev. But not for prod.

Reply:
Right, my other comment where you added the param should solve that. Essential we want that param with a default value of True. Looks like I typed false by mistake